packages/memprocfs.vm/tools/chocolateyinstall.ps1

```diff
@@ -3,8 +3,41 @@ Import-Module vm.common -Force -DisableNameChecking
 
 $toolName = 'MemProcFS'
 $category = VM-Get-Category($MyInvocation.MyCommand.Definition)
+# Get latest version from GitHub releases with a curl equivalent in powershell
```

Comment:
Use `Get-ChocolateyWebFile` chocolatey helper instead of curl, as [we do when we use signature verification](https://github.com/mandiant/VM-Packages/blob/8d2ee4f6704f7745b330dd4d5ac4f939885f321f/packages/sysinternals.vm/tools/chocolateyinstall.ps1#L15)